src/main/java/roomescape/domain/Reservation.java

```diff
@@ -0,0 +1,37 @@
+package roomescape.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+public class Reservation {
+    private long id;
+    private String name;
+    private LocalDateTime dateTime;
```

Comment:
final 로 선언해 재할당을 막아주세요!

```suggestion
    private final long id;
    private final String name;
    private final LocalDateTime dateTime;
```